src/encp.py

```diff
@@ -2521,6 +2521,9 @@ def check_library(library, e):
         #                               "%s is ignoring requests." % lib)
         #if state == "pause":
         #    status_ticket['status'] = (e_errors.PAUSE,
+        if state == "noread" and e.check:
+            # ignore LM state
+            status_ticket['status'] = (e_errors.OK, None)
         #                               "%s is paused." % lib)
         if state == "noread" and is_read(e):
```

Comment:
Should this be elsif? As written, it seems status_ticket['status'] will be overwritten (and thus the behavior will be unchanged) as long as (state == "noread" and is_read(e)), is that WAI?